### ISSUE TITLE
Documentation: provide a documentation about how this module work and what data is sent to GA

### ISSUE THREAD

**Issue by matks** [PrestaShop/ps_googleanalytics#74]
In order to be able to provide accurate and extended configuration capabilities / data synchronization between PS and GA, it would be useful to provide a documentation about how this module work and what data is sent to GA.

The doc should explain the inner working of the module.
The doc should explain what data are being sent from PS to GA, and where.

Once the list is complete, it'll provide an accurate overview of what data is missing / can be expanded (see https://github.com/PrestaShop/ps_googleanalytics/issues/57) and maybe open the path for advanced configuration settings (being able, from BO, to enable/disable what is being sent)

**Comment by Hlavtox** [comment]
In my opinion, the data the module should send is clearly defined in Google Analytics's documentation and the module should respect that.

**For example, the discussion regarding taxes.**
https://developers.google.com/analytics/devguides/collection/analyticsjs/enhanced-ecommerce#measuring-transactions

```
ga('ec:addProduct', {   
  'id': 'P12345',      
  'name': 'Android Warhol T-Shirt',
  'category': 'Apparel',     
  'brand': 'Google',        
  'variant': 'black',        
  'price': '29.20',  // Product price without tax
  'coupon': 'APPARELSALE',      
  'quantity': 1      
});

ga('ec:setAction', 'purchase', {     
  'id': 'T12345',  
  'affiliation': 'Google Store - Online', 
  'revenue': '37.39',  // Order total with tax
  'tax': '2.85',  // Order total tax
  'shipping': '5.34',  // Order total shipping with tax
  'coupon': 'SUMMER2013' 
});
```

**Comment by Kharny** [comment]
I agree with this issues. Actually I do no how to install the module to start. I did not find the information except configuration: 1 install the module.
